app/Events/CheckoutableCheckedOut.php

```diff
@@ -14,17 +14,25 @@ class CheckoutableCheckedOut
     public $checkedOutTo;
     public $checkedOutBy;
     public $note;
+    public $checkout_note;
+    public $total;
 
     /**
      * Create a new event instance.
      *
      * @return void
      */
-    public function __construct($checkoutable, $checkedOutTo, User $checkedOutBy, $note)
+    public function __construct($checkoutable, $checkedOutTo, User $checkedOutBy, $note, $checkout_qty=null, $checkout_note=null)
```

Comment:
I wonder if this `$checkout_qty` default should be `1`?

Reply:
fixed